src/util/system.cpp

```diff
@@ -1199,6 +1199,9 @@ int64_t GetStartupTime()
 
 fs::path AbsPathForConfigVal(const fs::path& path, bool net_specific)
 {
+    if (path.is_absolute()) {
+        return path;
+    }
```

Comment:
Would be nice if each commit had a test that failed before compilation and passes after. That might simplify review, no?

Reply:
Do tests enabled in 5124b1cef24a9b30876ddedf922dfdb555bd4ace do this job?

Reply:
Can they be enabled in this commit (the first commit)?

Reply:
Do you mean to combine the "Enable all tests in feature_config_args.py " commit (5124b1cef24a9b30876ddedf922dfdb555bd4ace) into the "Return absolute path early in AbsPathForConfigVal" commit (c21150fd6f598196fae93cf4bddc6e7625a43660)?

Master branch plus this combined commit will fail the test.
And the test will be passed on top of the 584621f9a81b6cc547c8f1365f1b358866e9a335 and b921f6ee951f0ad6d87eeeb372136a7314af0b0e commits.

Did I understand your intention correctly?